src/runtime_src/core/edge/drm/zocl/edge/zocl_edge_xclbin.c

```diff
@@ -272,7 +274,9 @@ zocl_xclbin_read_axlf(struct drm_zocl_dev *zdev, struct drm_zocl_axlf *axlf_obj,
 	 * Read AIE_RESOURCES section. aie_res will be NULL if there is no
 	 * such a section.
 	 */
+
 	zocl_read_sect(AIE_RESOURCES, &aie_res, axlf, xclbin);
+	zocl_read_sect(AIE_RESOURCES_BIN, &aie_res_bin, axlf, xclbin);
```

Comment:
I dont think, you can call this for RESOURCES_BIN. Instead you can check the existence of this section below